app.rb

```diff
@@ -4,17 +4,57 @@
 
 # First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
   # Explain how the data is structured
+  It's categorized by apartments and tenants. It has string value sand numerical values.
   # What are the properties for each of the two types of hashes
 
+
 # Use enumerables to -
   # Print all the addresses for the apartments
+      appartments[:address]
```

Comment:
This won't work for a few reasons:

1. `appartments` is mis-spelled (an extra *p*).
2. `apartments` is an array, and arrays don't have an `address` property.

Instead you have to do as you did below on line 15 using `each`